collector/common/system.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import "strings"
+
+const (
+	adminDB  = "admin"
+	configDB = "config"
+	localDB  = "local"
+)
+
+const sysCollPrefix = "system."
+
+// IsSystemDB tests whether system db name passed
+func IsSystemDB(dbName string) bool {
+	switch dbName {
+	case adminDB, configDB, localDB:
```

Comment:
If those strings are used only in once place, there is no point in replacing string literals with constants with the same names.

Reply:
Done